dakota/synergistic/fs-ty/fs-ty.in

```diff
@@ -0,0 +1,28 @@
+# Multidimensional parameter study 
```

Comment:
This file is quite strange. Is this what a dakota input file looks like?

Reply:
Yup it is.